test/test_helper.rb

```diff
@@ -28,6 +28,12 @@ class ActionDispatch::IntegrationTest
   end
 end
 
+class ActionMailer::MessageDelivery
```

Comment:
It would probably be better to change the tests to look for "enqueued" email deliveries, tbh, so that you're actually testing that deliver_later is called.